pom.xml

```diff
@@ -189,6 +189,11 @@ Konstanz, and KNIME GmbH.</license.copyrightOwners>
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>2.9.0</version>
```

Comment:
This should be `<scope>test</scope>`, no?

Reply:
Yes, thank you 👍